samples/bluetooth/mesh/mesh_dfu/target/README.rst

```diff
@@ -69,10 +69,9 @@ The models are used for the following purposes:
 * Config Server allows configurator devices to configure the node remotely.
 * Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
   These callbacks trigger blinking of the LEDs.
-* The Binary Large Object (BLOB) Transfer models, :ref:`zephyr:bluetooth_mesh_blob_srv` and :ref:`zephyr:bluetooth_mesh_blob_cli`, provide functionality for sending large binary objects from a single source to many Target nodes over the Bluetooth mesh network.
-  It is the underlying transport method for the Device Firmware Update (DFU).
-  BLOB Transfer Server is instantiated on the primary element.
-* The :ref:`zephyr:bluetooth_mesh_dfu_srv` model and its base models are instantiated on the secondary element.
+* Use of Binary Large Object (BLOB) Transfer models is the underlying transport method for the DFU.
+  BLOB Transfer Server is instantiated on the primary element and used to receive the firmware image binary from the Distributor node.
+* To implement the Target node functionality of the :ref:`zephyr:bluetooth_mesh_dfu` subsystem, the :ref:`zephyr:bluetooth_mesh_dfu_srv` model and its base models are instantiated on the primary element.
```

Comment:
This phrase belongs to distributor readme. It should replace line 85:
https://github.com/nrfconnect/sdk-nrf/blob/842167e23d0d6acb5efeb540b3db76b6bf004936/samples/bluetooth/mesh/mesh_dfu/distributor/README.rst?plain=1#L85

And line 84 in the distributor readme is not needed because FW Update Client is used for target nodes update and for self update:
https://github.com/nrfconnect/sdk-nrf/blob/842167e23d0d6acb5efeb540b3db76b6bf004936/samples/bluetooth/mesh/mesh_dfu/distributor/README.rst?plain=1#L84